src/parse.c

```diff
@@ -9012,7 +8995,8 @@ static void p_case_statement_alternative(tree_t stmt)
    consume(tASSOC);
 
    tree_t b = tree_new(T_SEQUENCE);
-   tree_set_ident(b, loc_to_ident(CURRENT_LOC));
+   // TODO: Do we really need identifier here??
+   //tree_set_ident(b, loc_to_ident(CURRENT_LOC));
```

Comment:
If the tests pass without this then it's unnecessary and you can just delete it.

Reply:
I still need to debug the tests, but the failing ones were the same with or without it.

Reply:
I debugged the 3 regression failures, only the expected name needed as you pointed out.

However, there is still failure in unit test `parse/guarded`. It complains about hiding `_P0` name, instead of
Q. However, IMHO `_P0` is not "hidden" since the surrounding block creates hierarchical separation and own scope. So, `_P0` will have its own hierarchical path. AFAIK you can't reffer to process in RHS the same way as to signal, so there is actually nothing to "hide".

Reply:
> 
> AFAIK you can't reffer to process in RHS the same way as to signal, so there is actually nothing to "hide".

Sure there is. E.g.

```vhdl
p: x <= '1';
b: block is
begin
  p: process is
  begin
    report p'instance_name;  -- What does P refer to?
  end process;
end block;
```

The problem here is we shouldn't insert statements with implicit names into the symbol table. That didn't cause a problem before because the `_lineXX` names were unique. `p_process_statement` already handles this but where `p_concurrent_signal_assignment_statement`, `p_concurrent_assertion_statement`, etc. do

```c
insert_name(nametab, conc, NULL);
```

needs to be

```c
if (label) insert_name(nametab, conc, NULL);
```

Reply:
OK, I see. There goes my VHDL knowledge :D The same should be the applied in `p_concurrent_procedure_call_statement`. With these, the regression passes.